cluster/cluster_SUITE.erl

```diff
@@ -188,6 +189,35 @@ acks(Config) ->
 
     Config.
 
+concurrent_fetch(_Config) ->
+    Topic = vg_test_utils:create_random_name(<<"cluster_concurrent_fetch">>),
+    RandomRecords = [crypto:strong_rand_bytes(60), crypto:strong_rand_bytes(60),
```

Comment:
note that using this instead of rand gates everything around a single lock in a nif, so best to use rand when you need things to go fast in many processes.  doesn't apply here, maybe.